lib/Services/Facebook.js

```diff
@@ -14,21 +14,38 @@ class Facebook {
     }
 
     async login ({ user, password, submit = false }) {
-        this.logger.info('Logging in...');
+		// this.logger.info('Logging in...');
         await this.driver.input('email', user);
         await this.driver.input('pass', password);
-        await this.driver.submit('#login_form');
-        this.logger.info(`Email: ${user} Password: ${password.replace(/./g, '*')}`);
+		await this.driver.click('#u_0_t');
+		// this.logger.info(`Email: ${user} Password: ${password.replace(/./g, '*')}`);
     }
 
     async getPhotos (quantity = 1) {
-        this.driver.navigate(this.url + '/photos');
+		this.driver.navigate(this.url + '/photos');
 
-        await this.driver.click('.uiMediaThumb', quantity);
-        await this.driver.click('a[data-action-type="open_options_flyout"]', 1);
-        await this.driver.click('a[data-action-type="download_photo"]', 1);
+		// Open spotlight
+		await this.driver.click('.uiMediaThumb');
+
+		// Get image src of all pictures
+		do {
+			try {
+				const elem = await this.driver.nodeService.get('.spotlight')
+				let alt = await elem[0].getAttribute('alt');
+				// A none empty alt means the hd image is loaded
+				while (alt == "") {
+					alt = await elem[0].getAttribute('alt');
+				}
+				// Print the image source url
+				console.log(await elem[0].getAttribute('src'))
+				// Display next image
+				await this.driver.click('.next')
+			} catch (e) {
+				// Catch error when fail then retry the same image
+				// this.logger.error(e)
+			}
+		} while (true)
 
```

Comment:
Do we need a better way of getting out of this loop? (I know how difficult this could potentially be 😅) when using this for yourself did you just let it run for x hours and assume it was finished?